contracts/utils/ICS02ClientUpgradeable.sol

```diff
@@ -83,19 +86,50 @@ abstract contract ICS02ClientUpgradeable is IICS02Client, IICS02ClientErrors, Ac
     )
         external
         returns (string memory)
+    {
+        string memory clientId = nextClientId();
+        _addClient(clientId, counterpartyInfo, client);
+        return clientId;
+    }
+
+    /// @inheritdoc IICS02Client
+    function addClient(
```

Comment:
Just had a thought about this... It seems a little impractical to have this called by a gov or a timelocked contract, right? It means that adding a new canonical light client would be a very slow process 🤔

Reply:
This is a role, not called by an admin...

Reply:
Right, but we would then need a separate multisig for these roles, right? This needs to be communicated so that we set it up in a useful way.
@Zygimantass FYI

Reply:
I think this can be just a wallet we control. No need for a multisig imo

Reply:
Sure! In any case, seems both solvable and a non-issue. Thanks for the clarifications :)